src/app/design-patterns/observer-rxjs/service/roulette.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@angular/core';
+import * as faker from 'faker';
+import { Subject } from 'rxjs';
+
+import { RouletteModel } from '../roulette.model';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class RouletteService {
+  generateRandomBets() {
+    const bet = {
+      amount: faker.finance.amount(0, 36),
+      avatar: faker.image.avatar(),
+      color: faker.commerce.color(),
```

Comment:
the color should be an enum(red, green, black)

Reply:
If it's a ENUM, how can it be random?

color: COLOR.RED || COLOR.GREEN || COLOR.BLACK?